pkgs/applications/networking/instant-messengers/element/element-desktop.nix

```diff
@@ -1,6 +1,8 @@
 { lib, fetchFromGitHub
 , makeWrapper, makeDesktopItem, mkYarnPackage
-, electron, element-web
+, electron_12, element-web
```

Comment:
Electron was updated to electron_13 on master. Reverting this to electron_12 will probably break Element's data if it was ever run on electron_13.

Reply:
Why can't we overwrite the element in inputs instead of changing multiple occurrences of it throughout the file?

Reply:
> Electron was updated to electron_13 on master. Reverting this to electron_12 will probably break Element's data if it was ever run on electron_13.


Downgrading a state from electron_12 to electron_13 works fine for me